### DISCUSSION TITLE
Remove rust-cpython support (at least from the new templates)?

### DISCUSSION THREAD

**Discussion by henryiii** [PyO3/maturin#2031] [General]
Currently, there's a list of options when starting a new project that includes `rust-cpython`. Upon inspection, it turns out [that `rust-cpython` was declared abandoned](https://github.com/dgrunwald/rust-cpython) 7 months ago (2+ years since last meaningful commit, not counting this notice) and doesn't support Python 3.12+. There's a note in the readme that projects should move to PyO3.

Shouldn't this be removed from the new project options, to simplify the user choice? Maybe the auto detection code could be removed too, but at least taking it out of the templates seems reasonable at this point. I have actually had someone ask me whether they should pick pyo3 or rust-python a while back because they saw it in this list.

**Comment by messense** [comment]
Removed from new templates in #2034

**Comment by messense** [comment]
Closing in favor of #2036.
